src/chromatix/functional/propagation.py

```diff
@@ -192,53 +195,25 @@ def transfer_propagate(
             for zero padding.
         kykx: If provided, defines the orientation of the propagation. Should
             be an array of shape `[2,]` in the format [ky, kx].
+        shift_yx: If provided, defines a shift in microns in the destination
+            plane. Should be an array of shape `[2,]` in the format `[y, x]`.
+        dx: If provided, defines a different output sampling at the output
+            plane.
+        N_out: If provided, defines the output shape of the field. Should be
+            a tuple of integers. If not provided and ``dx`` is provided, the
+            output shape will default to that of the input field.
         mode: Either "full" or "same". If "same", the shape of the output
             ``Field`` will match the shape of the incoming ``Field``. Defaults
             to "full", in which case the output shape will include padding.
     """
     field = pad(field, N_pad, cval=cval)
     propagator = compute_transfer_propagator(field, z, n, kykx)
-    field = kernel_propagate(field, propagator)
-    if mode == "same":
-        field = crop(field, N_pad)
-    return field
-
-
-def exact_propagate(
```

Comment:
This is a nice change! We should be loud about it in docs or changelogs somehow.

Reply:
Sure! Where is the changelog btw 🙈 I'm used to seeing it in the root of the repo as `CHANGELOG.md`

Reply:
I've just been keeping track in the GitHub releases so far but you're right 🙈